requirements.txt

```diff
@@ -4,7 +4,9 @@ coverage==4.1
 coveralls==1.1
 decorator==4.0.10
 dj-database-url==0.4.1
+dj-static==0.0.6
```

Comment:
The Admin site header is set and the index page title is still Platform administration.